### PR TITLE
Adjust for breaking change in tifffile 2021.10.10.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by cgohlke** [comment]
Sorry, this wasn't intended to break anything. I don't think I ever used that feature... I could release a new version of tifffile if you can provide a small test case of the API and cases you rely on.

**Comment by danielballan** [comment]
Thanks @cgohlke! I'm just catching up and you already fixed it. You are a hero!

**Comment by dylanmcreynolds** [comment]
So this PR is now not needed?

**Comment by danielballan** [comment]
Well, it's still worth having because the old name is deprecated and I think we want to support old versions of tifffile for awhile.

PIp doesn't always respect minimum version pins. It's getting better about that, but lab software environment often have old pip.

File: tiled/readers/tiff_sequence.py [hunk 1]
```diff
@@ -151,3 +151,17 @@ def macrostructure(self):
                 (shape[2],),
             ),
         )
+
+
+def _safe_asarray(seq, index=None):
+    """
+    Wrap FileSequence.asarray to cope with this brekaing change in tifffile.
```
Comment (clintonroy) [inline tiled/readers/tiff_sequence.py:158]: breaking

**Pull request merged**